package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
There shouldn't be a package.json outside of frontend. Also, Axios is unnecessary - we're using fetch everywhere else.

Reply:
We will remove package.json and axios.